python/pyspark/sql/udf.py

```diff
@@ -414,6 +415,9 @@ def _test():
         .appName("sql.udf tests")\
         .getOrCreate()
     globs['spark'] = spark
+    # Hack to skip the unit tests in register. These are currently being tested in proper tests.
+    # We should reenable this test once we completely drop Python 2.
+    del pyspark.sql.udf.UDFRegistration.register
```

Comment:
To doubly make sure, I tested and checked:

-  that it doesn't affect the main codes:

  ```
  >>> help(spark.udf.register)
  ```

  ```
  Help on method register in module pyspark.sql.udf:
  
  register(name, f, returnType=None) method of pyspark.sql.udf.UDFRegistration instance
  ...
  ```

- the generated doc too just to make sure.

- it works in main codes.

- the tests pass.